tests/js/client/http/api-gharial-spec.js

```diff
@@ -846,4 +846,37 @@ describe('_api/gharial testing cross-graph deletes', () => {
 
     expect(deleted).to.equal(true);
   });
+
+  it('should not create multiple orphan collection entries', () => {
```

Comment:
Can we remove this test from this PR?
It is contained in a different PR and is totally unreleated.